packages/core/src/components/accordion/accordion-item/accordion-item.tsx

```diff
@@ -50,13 +62,27 @@ export class TdsAccordionItem {
   }>;
 
   render() {
```

Comment:
Nice! What do you think about moving this into a function outside of render? So something like this:

```
/** Generate dynamic classes for the accordion item */
  private getClasses() {
    const shouldAnimate = this.animation !== 'none' && !this.initialLoad;

    return {
      'tds-accordion-item': true,
      'disabled': this.disabled,
      'expanded': this.expanded,
      [`tds-accordion-item-animation-open-${this.animation}`]: shouldAnimate && this.expanded,
      [`tds-accordion-item-animation-close-${this.animation}`]: shouldAnimate && !this.expanded,
    };
  }
```

And then in render:

`const classes = this.getClasses();`